### ISSUE TITLE
SSH Only option

### ISSUE THREAD

**Issue by AlexeyGorbushin** [pwn20wndstuff/Undecimus#1984]
Could you add the 'SSH Only' option functionality in the settings list as it was in at least in v4.3.1? It would be a great feature for researchers.

**Comment by badger200** [comment]
Yes this is crucial for getting root commandline access when things go south and the jailbreak fails to complete for various reasons. I can’t fathom why it would be removed.

**Comment by bigrck64** [comment]
I would also like this option to come back, like auto respring.

**Comment by SomethingGeneric** [comment]
+1'd

**Comment by badger200** [comment]
Protip for meanwhile: I had a situation where my iPad died from a full disk and would not boot past black screen white Apple logo.

I was able to solve it with a lot of tinkering, but the crucial tool was downloading “BootRai1n”, a bootable USB (or CD/DVD) Linux mini-OS with just enough to run checkra1n, which states that it is compatible to switch to and from other jailbreaks.

It worked to get me back into my device and I was able to proceed with unc0ver and my device is as if none of this ever happened. The checkra1n didn’t even seem to leave any new files on the file system. Nice!

Really saved my ass big time!!!!

**Comment by tausift0** [comment]
Yes please add it

**Comment by badger200** [comment]
Either @pwn20wndstuff or @sbingner tweeted that “Elucubratus” (whatever that is, it’s defined nowhere?) “is now open source”, and unc0ver was also claimed to be open source I thought? 

So where is the source so we can restore these features ourselves if the developers can’t/won’t do it in the past 11 months we’ve been asking for it?
